samples/core/src/test/resources/BillExportJobTest/expected/bills_pg.csv

```diff
@@ -0,0 +1,6 @@
+1,true,14.98 EUR,1.30 EUR,1,1,,1,Schnitzel-Menü,DELIVERED,6.99 EUR,mit Ketschup
```

Comment:
but... this has to be done by issuing the PR to that branch right? I can not merge this PR to that other branch...